docs/how-to-work-on-coding-challenges.md

```diff
@@ -211,11 +211,14 @@ Here are specific formatting guidelines for challenge text and examples:
 - Language keywords go in `<code>` tags. For example, HTML tag names or CSS property names
 - The first instance of a keyword when it's being defined, or general keywords (i.e. "object" or "immutable") go in `<dfn>` tags
 - References to code parts (i.e. function, method or variable names) should be wrapped in `<code>` tags. See example below:
-- Use <code>parseInt</code> to convert the variable <code>realNumber</code> into an integer.
-- Multi-line code blocks **must be preceded by an empty line**. The next line must start with three backticks followed immediately by one of the [supported languages](https://prismjs.com/#supported-languages). To complete the code block, you must start a newline which only has three backticks and **another empty line**.
-  **Note:** If you are going to use an example code in YAML, use `yaml` instead of `yml` for the language to the right of the backticks.
-
-See example below:
+```md
+Use <code>parseInt</code> to convert the variable <code>realNumber</code> into an integer.
+```
+- References to file names and path directories (i.e. `package.json`, `src/components`) should be wrapped in `<code>` tags.
+- Mention of tools, packages, and trademarks (i.e. _Pug, MongoDB_) should be wrapped in `<em>` tags.
```

Comment:
```suggestion
- Mention of tools, packages, and trademarks (e.g. _Pug, MongoDB_) should be wrapped in `<em>` tags.
```
Same nitpick.  Also, I didn't see any discussion about this change.  It seems fine to me, but I thought I'd highlight it since it didn't appear in the issue.

Reply:
These were good nitpicks. We could have some more discussion about this...

I would just say, these should **not** go within `code` tags. So, I would be happy removing this, or leaving it as is.

Reply:
I agree. If we can use semantic elements, why not?

I'll leave this unmerged for now so another reviewer can register an opinion, but I like the change.